### PR TITLE
RHMAP-15417 - Add Eslint into grunt test task

### PR DESCRIPTION
**Pull request by camilamacedo86**
https://issues.jboss.org/browse/RHMAP-15417

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['fh:test','eslint']);
```
Comment (wtrocki) [inline Gruntfile.js]: we used to have different order. eslint first. 
I see that there is unit task bellow with jshint - with that change it will be misleading what we using here.

**Comment by camilamacedo86** [comment]
@wtrocki Done.:-)

File: Gruntfile.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-jsxgettext');
   grunt.loadNpmTasks('grunt-zanata-js');
 
-  grunt.registerTask('test', ['fh:test']);
+  grunt.registerTask('test', ['eslint','fh:test']);
```
Comment (wtrocki) [inline Gruntfile.js]: I think that test is not standard task we support and use in CI. Basic tasks (backed by grunt plugin are unit and accept). This change should remove jshint

Comment (camilamacedo86) [inline Gruntfile.js]: @wtrocki But if we need change the standard of the GruntJS file and tasks need be in another PR / JIRA. This JIRA is a task which is a part of one epic to add Eslint in the project and call it in the test task.

**Comment by camilamacedo86** [comment]
Closing this PR in favor of the #322. It was centralized there.
